test/ziggurat/streams_test.clj

```diff
@@ -90,12 +90,12 @@
                                                          (assoc-in [:stream-router :default :application-id] (rand-application-id))
                                                          (assoc-in [:stream-router :default :oldest-processed-message-in-s] oldest-processed-message-in-s)
                                                          (assoc-in [:stream-router :default :changelog-topic-replication-factor] changelog-topic-replication-factor)))]
-    (Thread/sleep 5000)                                    ;;waiting for streams to start
+    (Thread/sleep 10000)                                    ;;waiting for streams to start
```

Comment:
is this the cause of the random failing?

Reply:
Yes. Seems like it just needed enough time to receive the messages.